src/lib/requests.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable max-len */
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the
+ Apache License, Version 2.0 (the 'License') and you may not use these files
+ except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop
+ files are distributed onan 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Raman Mangla <ramanmangla@google.com>
+ - Abhimanyu Kapur <abhi.kapur09@gmail.com>
+
+ --------------
+ ******/
+/* istanbul ignore file */
+// Testing will be covered in PR #11
+
+import { config } from './config'
+// @ts-ignore
+import { Logger, ThirdpartyRequests, BaseRequestConfigType } from '@mojaloop/sdk-standard-components'
+
+// Config file to instantiate ThirdPartyRequest object
+const configRequest: BaseRequestConfigType = {
+  dfspId: config.get('PARTICIPANT_ID') as string,
+  logger: new Logger(),
+  // TODO: Decide on below later
```

Comment:
Could you add comments here about whatever the TLS issue was? Maybe open a ticket and reference that here?

Reply:
Will we also have to update helm charts and other deployment config?

Reply:
good catch! will do the former
I'm not sure about the latter - @lewisdaly thoughts?